### DISCUSSION TITLE
Refreshing app but no resources are created

### DISCUSSION THREAD

**Discussion by joelmarty** [argoproj/argo-cd#18249] [Q&A]
Hi,

I have an application sharing a namespace with another (one has the service and its dependencies, and the other has the load-test for the app in the form of a `Job` resource).

Both apps sync from the same repo, but under different folders. The load-test manifests references resources managed under the other app (but since it's the same namespace, it should work, right?).

The service app works fine, but the load-test app does not show any resource pending sync when refreshing.

How can I debug/investigate why is that ? What are the most common causes for this issue ?

**Comment by ChristianCiach** [comment]
I think we need more information to make any assumptions about this. Maybe you could provide a minimal example to reproduce the issue?

**Reply by joelmarty** [reply]
I actually found out that the issue was because of a policy in the company: there is a requirement to provide PodDisruptionBudget resources but the policy was failing silently (no feedback whatsoever is given).
I don't believe this is an issue with ArgoCD though.
